dotcom-rendering/src/lib/useBridgetVersion.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from 'react';
+import { environmentClient } from './bridgetApi';
+
+export const useBridgetVersion = (): string | undefined => {
```

Comment:
I wonder if we should delete this module? I'm not sure when we're planning to use it.

Reply:
I think we should drop it. It's simple enough that the effort to reimplement if we need it is small.